plugins/go_replace_words.json

```diff
@@ -0,0 +1,7 @@
+{
+  "wasm": [
+    {
+      "path": "plugins/go_replace_words.wasm"
+    }
+  ]
+}
```

Comment:
Im committing the plugin binary used for api tests directly to the repo though there are some other options:
- build plugin from source (would require `go` and `tinygo`)
- use Rust to write test plugin (takes a long time to compile)
- load test plugin over http

Reply:
Compiling all examples in the existm rust-pdk repository to wasm takes 16 seconds for me.

Reply:
Right I pulled in `api_common` for the struct definitions which added various dependencies. It would be faster by using a hashmap instead like in golang. But still the plugin would have to be recompiled for scratch for every CI run, for something that shouldnt change at all. Anyway wasm is sandboxed so it cant do anything malicious. We could load the wasm file over HTTP from lemmy-plugins repo (like we are doing for pictrs), but that also wouldnt give any real benefit.

Reply:
Done